### PR TITLE
Scripts folder and add_multiple_devices helper script

### PR DESCRIPTION
**Pull request by marciogranzotto**

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        --deviceName The name of the device. This will be a prefix of the actual name. I.e: 'Test Device 1', 'Test Device 2'...
+        --location Device Location (Bedroom, Living Room, etc.)
+        --template The device template name to be created (RCF84, A1, etc.)
+        --quantity The amount of devices that will be created
+```
+
+### Example
+```bash
+# On the repository root folder
+./scripts/add_multiple_devices.sh --bondID ZPEA77140 --deviceName "Test Shade" --location "Testing" --template RMS12 --quantity 15
```
Comment (endy-s) [inline scripts/README.md]: I believe it is safer to use "generic Bond IDs"
```suggestion
./scripts/add_multiple_devices.sh --bondID ZPEA12345 --deviceName "Test Shade" --location "Testing" --template RMS12 --quantity 15
```

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Scripts
+
+Here you can find some useful scripts to automate tasks using the `bond-cli` tool.
+
+## [add_multiple_devices.sh](add_multiple_devices.sh)
+A script to add multiple copies of the same device. Useful for testing.
```
Comment (endy-s) [inline scripts/README.md]: The "copy" doesn't feel right...
```suggestion
A script to add multiple devices with the same template, location, and name prefix. Useful for testing.
```

File: scripts/add_multiple_devices.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+helpFunction()
+{
+  echo ""
+  echo "Usage: $0 --bondID ZXBL00000 --deviceName 'Test Device' --location 'Bedroom' --template RMS12 --quantity 10"
+  echo -e "\t--bondID The ID of the Bond you want to create the new devices"
+  echo -e "\t--deviceName The name of the device. This will be a prefix of the actual name. I.e: 'Test Device 1', 'Test Device 2'..."
+  echo -e "\t--location Device Location (Bedroom, Living Room, etc.)"
```
Comment (endy-s) [inline scripts/add_multiple_devices.sh]: Kep the pattern
```suggestion
  echo -e "\t--location The device location (Bedroom, Living Room, etc.)"
```

File: scripts/add_multiple_devices.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+helpFunction()
+{
+  echo ""
+  echo "Usage: $0 --bondID ZXBL00000 --deviceName 'Test Device' --location 'Bedroom' --template RMS12 --quantity 10"
+  echo -e "\t--bondID The ID of the Bond you want to create the new devices"
+  echo -e "\t--deviceName The name of the device. This will be a prefix of the actual name. I.e: 'Test Device 1', 'Test Device 2'..."
```
Comment (endy-s) [inline scripts/add_multiple_devices.sh:8]: Maybe you can change it to `device name prefix`?

**Review by endy-s — DISMISSED** [review]

**Comment by marciogranzotto** [comment]
@chrismerck can we merge this?

**Comment by chrismerck** [comment]
OK. But maybe add a README in the new scripts dir saying UNSUPPROTED / use at your own risk / YMMV... we are "supporting" the cli tool, but the scripts are face-value...

**Review by chrismerck — DISMISSED** [review]

File: scripts/add_multiple_devices.sh [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+bond discover
+bond select $bondid
+
+echo
+echo "Adding $deviceName (1 to $quantity) with template $template to $bondID"
+
+for (( i=1; i<=$quantity; i++ )); do
+  name="$deviceName $(printf %0*d ${#quantity} $i)"
+  bond device_create --name "$name" --location "$location" --template "$template"
```
Comment (fredericojordan) [inline scripts/add_multiple_devices.sh]: With the recent changes, this should be updated:
```suggestion
  bond devices create --name "$name" --location "$location" --template "$template"
```

**Review by fredericojordan — APPROVED** [review]

**Pull request merged**